presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -114,6 +114,7 @@
     private int fileSystemMaxCacheSize = 1000;
 
     private boolean writesToNonManagedTablesEnabled;
+    private boolean tableStatisticsEnabled;
```

Comment:
yep, we expect it to be disabled by default. @nezihyigitbasi do you know how could I fix product test?